index.js

```diff
@@ -69,7 +69,8 @@ var supportedTargets = [
   {runtime: 'electron', target: '3.0.0', abi: '64', lts: false},
   {runtime: 'electron', target: '4.0.0', abi: '64', lts: false},
   {runtime: 'electron', target: '4.0.4', abi: '69', lts: false},
-  {runtime: 'electron', target: '5.0.0', abi: '70', lts: false}
+  {runtime: 'electron', target: '5.0.0', abi: '70', lts: false},
+  {runtime: 'electron', target: '6.0.0', abi: '73', lts: false}
```

Comment:
`6.0.0` is still in beta, it should be added to `futureTargets` below

Reply:
10-4